### PR TITLE
Saved data - animation 

### PR DESCRIPTION
**Pull request by SavageWilliam**
#72

Following @ishiar feedback, ladder is only visualised when clicking complete

### CODE REVIEW

File: css/create-ladder.css [hunk 1]
```diff
@@ -87,9 +87,9 @@ body {
 }
 
 #ladder .fold {
-  border-left: 1rem solid #3B3561;
+  /*border-left: 1rem solid #3B3561;
```
Comment (des-des) [inline css/create-ladder.css]: :fire:

File: create-ladder.js [hunk 2]
```diff
@@ -18,6 +19,19 @@
...
 
+  const showBorders = () => {
+    const folds = document.querySelectorAll('.fold');
+    addBorder(folds,0);
+  }
+
+  //Add Border to folds
+  const addBorder = (folds) =>{
+    folds.forEach( fold => {
+      fold.classList += ' border';
```
Comment (des-des) [inline create-ladder.js]: use `classList.add`

File: create-ladder.js [hunk 3]
```diff
@@ -50,13 +63,13 @@
     const cloudM = document.getElementById('cloudM');
     cloudM.style.display ='none'
     const cloudTitle = document.getElementById('cloudTitle');
-    cloudTitle.className += ' animated pulse move-up'
+    cloudTitle.className += ' animated pulse move-up';
```
Comment (des-des) [inline create-ladder.js]: again `classList.add` is better, as it will prevent duplication

**Comment by SavageWilliam** [comment]
@des-des To avoid chaining .add() where I am adding multiple classes I;ve stuck to the className += method.. But changed to classList.add otherwise! Thanks

**Pull request merged**